packages/create-mc-app/src/tasks/download-template.js

```diff
@@ -65,13 +65,26 @@ module.exports = function downloadTemplate(options) {
             const sanitizedProjectDirectoryPath = options.projectDirectoryPath
               // Escape white spaces
               .replace(/ /g, '\\ ');
-            const result = await execa(
-              'mv',
```

Comment:
I suggest to simply add a ternary here:

```
process.platform == 'win32' || process.platform == 'cygwin' ? 'move' : 'mv'
```

Reply:
fair enough, initially i was using a totally different command with different params hence the if, but `move` is the same as `mv` so not needed anymore.